pallets/sto/src/lib.rs

```diff
@@ -282,7 +282,8 @@ decl_module! {
             let offering_amount: T::Balance = tiers
                 .iter()
                 .map(|t| t.total)
-                .fold(0.into(), |total, x| total + x);
+                .try_fold(0.into(), |total: T::Balance, x| total.checked_add(&x))
```

Comment:
What made `total: T::Balance` necessary when you already have `let offering_amount: T::Balance`?

Reply:
Yes, it is necessary at least in rustc-1.48.0-nightly